src/components/NavBar.svelte

```diff
@@ -1,38 +1,51 @@
 <script lang="ts">
     import { createEventDispatcher } from 'svelte';
     import Sidebar from './Sidebar.svelte';
+    import img from '$lib/image/Nav_Bar_Logo_For_StarWars_Franchise.jpeg';
```

Comment:
Would it be possible to fetch these images from the API?

Reply:
No these are images i generated & downloaded myself.